### ISSUE TITLE
Doubt regarding train test split

### ISSUE THREAD

**Issue by pavantadinada** [shreekantgosavi/Heart-Disease-Prediction-using-Machine-Learning#1]
from sklearn.model_selection import train_test_split

predictors = dataset.drop("target",axis=1)
target = dataset["target"]

X_train,X_test,Y_train,Y_test = train_test_split(predictors,target,test_size=0.20,random_state=0)

from the above code you wrote predictors and please explain me in detail about predictors i mean which attributes comes under this predictors.

**Comment by g-shreekant** [comment]
All except the target variable
